### PR TITLE
New ON List with Norms_MorenoMartinez2012

### PR DESCRIPTION
**Pull request by alzkuc**
Hi Mattis! Here is a new list I mapped. :-) @LinguList 

- [x] add new concept list
- [x] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

...

### CODE REVIEW

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -441,4 +441,5 @@ Smith-2017-679	Smith, Alexander D.	2017		679	basic	English	Burneo languages		Smi
...
+Snodgrass-1980-260	Snodgrass, Joan G. and Vanderwart, Mary	1980		260	basic, naming test	English	global 	https://psycnet.apa.org/doi/10.1037/0278-7393.6.2.174	Snodgrass1980		This list comprises 260 concepts from several widely studied semantic categories that are to be elicited using a standardized set of 260 pictures for use in experiments investigating differences and similarities in the processing of pictures and words. The pictures are black-and-white line drawings executed according to a set of rules that provide consistency of pictorial representation.
+MorenoMartinez-2012-360	Moreno-Martinez, Francisco J. and Montoro, Pedro R.	2012		360	naming test	English, Spanish	global	https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0037527&type=printable	MorenoMartinez2012		This work presents a new set of 360 high quality color images belonging to 23 semantic subcategories. These were named by 230 Spanish native speakers and come with multiple psycholinguistic variables, here: AoA, familiarity, typicality, visual complexity and lexical frequency.
```
Comment (LinguList) [inline concepticondata/conceptlists.tsv:445]: The URL points to the PDF of the article, but the URL is usually used to link the original data. Do you happen to have the data as well, a link to it? So I'd suggest to replace the link.

File: concepticondata/conceptlists/MorenoMartinez-2012-360.tsv [hunk 2]
```diff
@@ -0,0 +1,362 @@
+ID	NUMBER	ENGLISH	SPANISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	SEMANTIC_CATEGORY_23	AoA_M	FAMILIARITY_LICKERT_M	TYPICALITY_LICKERT_M	VISUALCOMPLEX_LICKERT_M	LEXICALFREQ_LOG
+MorenoMartinez-2012-360-1	1	"
```
Comment (LinguList) [inline concepticondata/conceptlists/MorenoMartinez-2012-360.tsv:2]: There's a newline that you introduced in the data, probably due to the use of `"`. I think you accidentally added the newline in the word `Armadillo`.

**Review by LinguList — COMMENTED** [review]
The CSVW seems fine to me, also by specifying float as the datatype. What does "LICKERT" mean in this context?

Maybe, if you introduce the variables, you might want to also point to them in your text presenting the concept list? 

NOte also that Language should not only be English, but also Spanish, right?

**Comment by alzkuc** [comment]
> The CSVW seems fine to me, also by specifying float as the datatype. What does "LICKERT" mean in this context?
> 
> Maybe, if you introduce the variables, you might want to also point to them in your text presenting the concept list?
> 
> NOte also that Language should not only be English, but also Spanish, right?

@LinguList good. I am glad. :-) Do you mean in the .json? I did specify Spanish there too. I think your idea is better - I will point to the specific variables in the text about the concept list and simplify the names of the variables.

File: concepticondata/conceptlists.tsv (hunk 1 above)
Comment (alzkuc) [reply concepticondata/conceptlists.tsv]: swapped for a link that points to the data and norms :)

**Comment by LinguList** [comment]
The file conceptlists.tsv contains one field for the language in which the glosses are given and one for the target languages, the target languages are not that interesting mostly, because it si also quite general (we often write "global"). But the source language is important, since it is interpreted on the website, if you do not write Spanish, the spanish gloss won't appear openly on the website.

**Comment by LinguList** [comment]
I meant the filed "SOURCE_LANGUAGE". Here, write `English, Spanish` (and maybe compare with other instances for examples).

**Comment by alzkuc** [comment]
> I meant the filed "SOURCE_LANGUAGE". Here, write `English, Spanish` (and maybe compare with other instances for examples).

Ja, genau, das hab ich dort so geschrieben - in der conceptlists.tsv Datei ist bei "SOURCE_LANGUAGE" English, Spanish. :-)

**Review by alzkuc — COMMENTED** [review]
review for merge

**Comment by alzkuc** [comment]
@LinguList diesen Pull Request kann ich leider nicht alleine mergen, es braucht eine "approving review" von jemandem, der/die nicht der/die Autor/in ist. Könntest du es bitte, wenn du mal Zeit dafür hast, machen? :-) vielen lieben Dank.

**Review by LinguList — APPROVED** [review]

**Comment by LinguList** [comment]
Yep, I have to formally approve it or somebody else. I just checked the data and think the mapping is fine and approved it.

**Comment by alzkuc** [comment]
> Yep, I have to formally approve it or somebody else. I just checked the data and think the mapping is fine and approved it.

thank you!

**Pull request merged**